### PR TITLE
Updated postman env_var template

### PR DESCRIPTION
**Pull request by Fabian-t800**
Updated the template

### CODE REVIEW

File: Environment Variables/Spotify_API.postman_environment_template.JSON.json [hunk 1]
```diff
@@ -59,9 +64,14 @@
 		},
 		{
 			"key": "refresher_token",
-			"value": "",
+			"value": "0",
```
Comment (cspireasv) [inline Environment Variables/Spotify_API.postman_environment_template.JSON.json]: Is it necessary to set the property value to "0"?

Comment (Fabian-t800) [reply Environment Variables/Spotify_API.postman_environment_template.JSON.json]: Property value doesn't need to be set to 0 for the refresher token.
It was thought out as a placeholder, signifying that it requires your own value, and should not be left empty.

**Review by nicolae-chedea — COMMENTED** [review]
remove one of the .json extensions from the file name.

**Review by DanDaika — APPROVED** [review]
I made modifications on this branch and push them, just to see what will happen.

**Pull request merged**